--- papermerge/core/checks.py.orig	2020-08-09 12:08:34.945343495 +0000
+++ papermerge/core/checks.py	2020-08-09 12:10:22.225071279 +0000
@@ -36,9 +36,9 @@
     """
 
     msg = {
-        "tesseract": "Without it, OCR of the documents is impossible",
+        "tessract": "Without it, OCR of the documents is impossible",
         "pdfinfo": "Without it, Papermerge won't function properly",
-        "pdftk": "Without it, Papermerge won't be able to cut/paste PDF pages"
+        "stapler": "Without it, Papermerge won't be able to cut/paste PDF pages"
     }
     error = "Papermerge can't find {}. {}."
     hint = "Either it's not in your PATH or it's not installed."
