--- papermerge/core/management/commands/worker.py.orig	2020-08-09 07:51:57.296933889 +0000
+++ papermerge/core/management/commands/worker.py	2020-08-09 08:26:17.898787470 +0000
@@ -100,7 +100,11 @@
     help = """Async Celery Worker"""
 
     def add_arguments(self, parser):
-        pass
+        parser.add_argument(
+            '--pidfile',
+            type=str,
+            help='Optional file used to store the process pid.\nThe program won’t start if this file already exists and the pid is still alive.'
+        )
 
     def handle(self, *args, **options):
         celery_app.config_from_object(
@@ -117,4 +121,8 @@
             quiet=True,
             concurrency=1
         )
+
+        if options['pidfile']:
+            celery_worker.pidfile = options['pidfile']
+
         celery_worker.start()
