--- mglib/pdfinfo.py.orig	2020-08-11 11:15:51.034679971 +0000
+++ mglib/pdfinfo.py	2020-08-11 11:22:44.209394583 +0000
@@ -13,7 +13,7 @@
 
 def get_tiff_pagecount(filepath):
     cmd = [
-        "/usr/bin/identify",
+        "identify",
         "-format",
         "%n\n",
         filepath
@@ -21,9 +21,10 @@
     compl = subprocess.run(
         cmd,
         stdout=subprocess.PIPE,
-        stderr=subprocess.PIPE
+        stderr=subprocess.PIPE,
+        env=os.environ.copy()
     )
-
+    logger.info("returncode: %s", compl.returncode)
     if compl.returncode:
 
         logger.error(
@@ -79,13 +80,15 @@
 
     # pdfinfo "${PDFFILE}" | grep Pages
 
-    cmd = ["/usr/bin/pdfinfo", filepath]
+    cmd = ["pdfinfo", filepath]
     compl = subprocess.run(
         cmd,
         stdout=subprocess.PIPE,
-        stderr=subprocess.PIPE
+        stderr=subprocess.PIPE,
+        env=os.environ.copy()
     )
-
+    logger.info("returncode: %s", compl.returncode)
+    logger.info("env: %s", os.environ.copy())
     if compl.returncode:
 
         logger.error(
